spec/models/state_file_md_intake_spec.rb

```diff
@@ -329,4 +329,68 @@
       end
     end
   end
+
+  describe "#sum_two_1099_r_followup_types_for_filer" do
+    context "with followups present" do
+      let!(:intake) { create(:state_file_md_intake, :with_spouse) }
+      let(:state_file_md1099_r_followup_with_one_followup_criterion_for_primary) do
+        create(
+          :state_file_md1099_r_followup,
+          income_source: "pension_annuity_endowment",
+          state_file1099_r: create(:state_file1099_r, taxable_amount: 1_000, intake: intake, recipient_ssn: intake.primary.ssn)
+        )
+      end
+      let!(:state_file_md1099_r_followup_with_both_followup_criteria_for_spouse_1) do
+        create(
+          :state_file_md1099_r_followup,
+          income_source: "pension_annuity_endowment",
+          service_type: "public_safety",
+          state_file1099_r: create(:state_file1099_r, taxable_amount: 1_000, intake: intake, recipient_ssn: intake.spouse.ssn)
+        )
+      end
+      let!(:state_file_md1099_r_followup_with_both_followup_criteria_for_spouse_2) do
+        create(
+          :state_file_md1099_r_followup,
+          income_source: "pension_annuity_endowment",
+          service_type: "public_safety",
+          state_file1099_r: create(:state_file1099_r, taxable_amount: 1_500, intake: intake, recipient_ssn: intake.spouse.ssn)
+        )
+      end
+
+      context "when only one followup is present" do
+        it "returns 0" do
+          expect(intake.sum_two_1099_r_followup_types_for_filer(:primary, :income_source_pension_annuity_endowment?, :service_type_public_safety? )).to eq(0)
+        end
+      end
+
+      context "when both followups are present" do
```

Comment:
sorry if i'm missing something but it doesn't look like there's a difference in the setup between this block and the one on line 360, though it says that one has only one follow-up and the other has 2?

Reply:
oh i see now, it's not that the followups are _present_, it's that they qualify them for the subtraction. i think i'd rename these contexts to "when the income source qualifies them" (or doesn't) or "when the followup has a qualifying service type" (or doesn't)